src/storage.js

```diff
@@ -0,0 +1,12 @@
+const storageKey = (userName) => `${userName}Todos`;
+
+export const getStoredTodos = (userName) => {
+  if (localStorage.getItem(storageKey(userName)) === null) {
+    return [];
+  }
+  return JSON.parse(localStorage.getItem(storageKey(userName)));
+};
+
+export const persist = (todoItem, userName) => {
```

Comment:
I think the first parameter is awaiting for _all_ the todo items to be stored in `localStorage`, isn't it?

it'd be better to rename it from `todoItem` => `todoItems`

```suggestion
export const persist = (todoItems, userName) => {
```